examples/doc-sync/fsync/fsync.js

```diff
@@ -0,0 +1,121 @@
+const WebRTCStar = require('libp2p-webrtc-star');
+const wrtc = require('wrtc');
+const {createClient} = require("@functionland/fula");
+const fs=require('fs');
+const { File } =  require('@web-std/file');
+const uuid = require('uuid');
+const yargs = require('yargs')
+var _ = require('lodash');
+
+
+const argv = require('yargs/yargs')(process.argv.slice(2))
+      .usage('Usage: $0 -i [input-file]')
+      .demandOption(['i'])
+      .argv;
+
+const DOC_PATH = argv.i
+
+if(!fs.existsSync(DOC_PATH)) {
+  console.error(`Input file ${DOC_PATH} does not exist.`)
+  process.exit(1)
+}
+
+const BOX_ID = process.env.BOX_ID
```

Comment:
If you have argv you can also pass the boxId using arguments.

Reply:
nice idea! will add this

Reply:
should be fixed now